reputation/types.proto

```diff
@@ -43,7 +43,7 @@ message PeerToPeerTrust {
 
 // Global trust level to NeoFS node.
 message GlobalTrust {
-    // Message format version. Effectively the version of API library used to create
+    // Message format version. The API library version effectively used to create
```

Comment:
i think there could happen a misunderstanding: the idea of the comment was: "That field means the version of that message. In fact (usually/in practice), the version of the API library (which was used to create that message) is used here."

please, check that your version brings the same meaning

Reply:
Got it.
That's a good example why commas are very important sometimes.